<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>367</width>
    <height>545</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Compass Toolkit</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <property name="spacing">
         <number>6</number>
        </property>
        <property name="margin">
         <number>0</number>
        </property>
        <item>
         <layout class="QFormLayout" name="formLayout">
          <property name="fieldGrowthPolicy">
           <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
          </property>
          <item row="4" column="1">
           <layout class="QGridLayout" name="gridLayout_2">
            <item row="5" column="2">
             <widget class="QSpinBox" name="intelligentOrientationSpinBox">
              <property name="minimum">
               <number>-180</number>
              </property>
              <property name="maximum">
               <number>180</number>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QRadioButton" name="headingLockRadioButton">
              <property name="text">
               <string>Headling Lock</string>
              </property>
              <property name="autoExclusive">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="flightModesLabel">
              <property name="text">
               <string>Flight Modes</string>
              </property>
             </widget>
            </item>
            <item row="5" column="0">
             <widget class="QSpinBox" name="headingLockSpinBox">
              <property name="minimumSize">
               <size>
                <width>20</width>
                <height>0</height>
               </size>
              </property>
              <property name="minimum">
               <number>-180</number>
              </property>
              <property name="maximum">
               <number>180</number>
              </property>
             </widget>
            </item>
            <item row="6" column="2">
             <widget class="QPushButton" name="intelligentOrientationCurrentPushButton">
              <property name="text">
               <string>Current</string>
              </property>
             </widget>
            </item>
            <item row="2" column="2">
             <widget class="QRadioButton" name="intelligentOrientationRadioButtton">
              <property name="text">
               <string>Intelligent Orientation</string>
              </property>
              <property name="autoExclusive">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item row="4" column="2">
             <widget class="QDial" name="intelligentOrientationDial">
              <property name="layoutDirection">
               <enum>Qt::LeftToRight</enum>
              </property>
              <property name="minimum">
               <number>-180</number>
              </property>
              <property name="maximum">
               <number>180</number>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="invertedAppearance">
               <bool>false</bool>
              </property>
              <property name="invertedControls">
               <bool>false</bool>
              </property>
              <property name="wrapping">
               <bool>true</bool>
              </property>
              <property name="notchesVisible">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item row="6" column="0">
             <widget class="QPushButton" name="headingLockCurrentPushButton">
              <property name="text">
               <string>Current</string>
              </property>
             </widget>
            </item>
            <item row="4" column="0">
             <widget class="QDial" name="headingLockDial">
              <property name="minimum">
               <number>-180</number>
              </property>
              <property name="maximum">
               <number>180</number>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="invertedAppearance">
               <bool>false</bool>
              </property>
              <property name="wrapping">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="2" column="1">
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="0" column="1">
           <layout class="QGridLayout" name="gridLayout_3">
            <item row="1" column="0">
             <widget class="QLabel" name="compassLabel">
              <property name="text">
               <string>Compass</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QTextEdit" name="compassTextEdit"/>
            </item>
            <item row="4" column="0">
             <layout class="QGridLayout" name="gridLayout_5">
              <item row="0" column="2">
               <widget class="QLCDNumber" name="headingLcdNumber">
                <property name="palette">
                 <palette>
                  <active>
                   <colorrole role="WindowText">
                    <brush brushstyle="SolidPattern">
                     <color alpha="255">
                      <red>0</red>
                      <green>0</green>
                      <blue>0</blue>
                     </color>
                    </brush>
                   </colorrole>
                   <colorrole role="Button">
                    <brush brushstyle="SolidPattern">
                     <color alpha="255">
                      <red>85</red>
                      <green>170</green>
                      <blue>255</blue>
                     </color>
                    </brush>
                   </colorrole>
                   <colorrole role="Light">
                    <brush brushstyle="SolidPattern">
                     <color alpha="255">
                      <red>213</red>
                      <green>234</green>
                      <blue>255</blue>
                     </color>
                    </brush>
                   </colorrole>
                   <colorrole role="Midlight">
                    <brush brushstyle="SolidPattern">
                     <color alpha="255">
                      <red>149</red>
                      <green>202</green>
                      <blue>255</blue>
                     </color>
                    </brush>
                   </colorrole>
                   <colorrole role="Dark">
                    <brush brushstyle="SolidPattern">
                     <color alpha="255">
                      <red>42</red>
                      <green>85</green>
                      <blue>127</blue>
                     </color>
                    </brush>
                   </colorrole>
                   <colorrole role="Mid">
                    <brush brushstyle="SolidPattern">
                     <color alpha="255">
                      <red>56</red>
                      <green>113</green>
                      <blue>170</blue>
                     </color>
                    </brush>
                   </colorrole>
                   <colorrole role="Text">
                    <brush brushstyle="SolidPattern">
                     <color alpha="255">
                      <red>0</red>
                      <green>0</green>
                      <blue>0</blue>
                     </color>
                    </brush>
                   </colorrole>
                   <colorrole role="BrightText">
                    <brush brushstyle="SolidPattern">
                     <color alpha="255">
                      <red>255</red>
                      <green>255</green>
                      <blue>255</blue>
                     </color>
                    </brush>
                   </colorrole>
                   <colorrole role="ButtonText">
                    <brush brushstyle="SolidPattern">
                     <color alpha="255">
                      <red>0</red>
                      <green>0</green>
                      <blue>0</blue>
                     </color>
                    </brush>
                   </colorrole>
                   <colorrole role="Base">
                    <brush brushstyle="SolidPattern">
                     <color alpha="255">
                      <red>255</red>
                      <green>255</green>
                      <blue>255</blue>
                     </color>
                    </brush>
                   </colorrole>
                   <colorrole role="Window">
                    <brush brushstyle="SolidPattern">
                     <color alpha="255">
                      <red>85</red>
                      <green>170</green>
                      <blue>255</blue>
                     </color>
                    </brush>
                   </colorrole>
                   <colorrole role="Shadow">
                    <brush brushstyle="SolidPattern">
                     <color alpha="255">
                      <red>0</red>
                      <green>0</green>
                      <blue>0</blue>
                     </color>
                    </brush>
                   </colorrole>
                   <colorrole role="AlternateBase">
                    <brush brushstyle="SolidPattern">
                     <color alpha="255">
                      <red>170</red>
                      <green>212</green>
                      <blue>255</blue>
                     </color>
                    </brush>
                   </colorrole>
                   <colorrole role="ToolTipBase">
                    <brush brushstyle="SolidPattern">
                     <color alpha="255">
                      <red>255</red>
                      <green>255</green>
                      <blue>220</blue>
                     </color>
                    </brush>
                   </colorrole>
                   <colorrole role="ToolTipText">
                    <brush brushstyle="SolidPattern">
                     <color alpha="255">
                      <red>0</red>
                      <green>0</green>
                      <blue>0</blue>
                     </color>
                    </brush>
                   </colorrole>
                  </active>
                  <inactive>
                   <colorrole role="WindowText">
                    <brush brushstyle="SolidPattern">
                     <color alpha="255">
                      <red>0</red>
                      <green>0</green>
                      <blue>0</blue>
                     </color>
                    </brush>
                   </colorrole>
                   <colorrole role="Button">
                    <brush brushstyle="SolidPattern">
                     <color alpha="255">
                      <red>85</red>
                      <green>170</green>
                      <blue>255</blue>
                     </color>
                    </brush>
                   </colorrole>
                   <colorrole role="Light">
                    <brush brushstyle="SolidPattern">
                     <color alpha="255">
                      <red>213</red>
                      <green>234</green>
                      <blue>255</blue>
                     </color>
                    </brush>
                   </colorrole>
                   <colorrole role="Midlight">
                    <brush brushstyle="SolidPattern">
                     <color alpha="255">
                      <red>149</red>
                      <green>202</green>
                      <blue>255</blue>
                     </color>
                    </brush>
                   </colorrole>
                   <colorrole role="Dark">
                    <brush brushstyle="SolidPattern">
                     <color alpha="255">
                      <red>42</red>
                      <green>85</green>
                      <blue>127</blue>
                     </color>
                    </brush>
                   </colorrole>
                   <colorrole role="Mid">
                    <brush brushstyle="SolidPattern">
                     <color alpha="255">
                      <red>56</red>
                      <green>113</green>
                      <blue>170</blue>
                     </color>
                    </brush>
                   </colorrole>
                   <colorrole role="Text">
                    <brush brushstyle="SolidPattern">
                     <color alpha="255">
                      <red>0</red>
                      <green>0</green>
                      <blue>0</blue>
                     </color>
                    </brush>
                   </colorrole>
                   <colorrole role="BrightText">
                    <brush brushstyle="SolidPattern">
                     <color alpha="255">
                      <red>255</red>
                      <green>255</green>
                      <blue>255</blue>
                     </color>
                    </brush>
                   </colorrole>
                   <colorrole role="ButtonText">
                    <brush brushstyle="SolidPattern">
                     <color alpha="255">
                      <red>0</red>
                      <green>0</green>
                      <blue>0</blue>
                     </color>
                    </brush>
                   </colorrole>
                   <colorrole role="Base">
                    <brush brushstyle="SolidPattern">
                     <color alpha="255">
                      <red>255</red>
                      <green>255</green>
                      <blue>255</blue>
                     </color>
                    </brush>
                   </colorrole>
                   <colorrole role="Window">
                    <brush brushstyle="SolidPattern">
                     <color alpha="255">
                      <red>85</red>
                      <green>170</green>
                      <blue>255</blue>
                     </color>
                    </brush>
                   </colorrole>
                   <colorrole role="Shadow">
                    <brush brushstyle="SolidPattern">
                     <color alpha="255">
                      <red>0</red>
                      <green>0</green>
                      <blue>0</blue>
                     </color>
                    </brush>
                   </colorrole>
                   <colorrole role="AlternateBase">
                    <brush brushstyle="SolidPattern">
                     <color alpha="255">
                      <red>170</red>
                      <green>212</green>
                      <blue>255</blue>
                     </color>
                    </brush>
                   </colorrole>
                   <colorrole role="ToolTipBase">
                    <brush brushstyle="SolidPattern">
                     <color alpha="255">
                      <red>255</red>
                      <green>255</green>
                      <blue>220</blue>
                     </color>
                    </brush>
                   </colorrole>
                   <colorrole role="ToolTipText">
                    <brush brushstyle="SolidPattern">
                     <color alpha="255">
                      <red>0</red>
                      <green>0</green>
                      <blue>0</blue>
                     </color>
                    </brush>
                   </colorrole>
                  </inactive>
                  <disabled>
                   <colorrole role="WindowText">
                    <brush brushstyle="SolidPattern">
                     <color alpha="255">
                      <red>42</red>
                      <green>85</green>
                      <blue>127</blue>
                     </color>
                    </brush>
                   </colorrole>
                   <colorrole role="Button">
                    <brush brushstyle="SolidPattern">
                     <color alpha="255">
                      <red>85</red>
                      <green>170</green>
                      <blue>255</blue>
                     </color>
                    </brush>
                   </colorrole>
                   <colorrole role="Light">
                    <brush brushstyle="SolidPattern">
                     <color alpha="255">
                      <red>213</red>
                      <green>234</green>
                      <blue>255</blue>
                     </color>
                    </brush>
                   </colorrole>
                   <colorrole role="Midlight">
                    <brush brushstyle="SolidPattern">
                     <color alpha="255">
                      <red>149</red>
                      <green>202</green>
                      <blue>255</blue>
                     </color>
                    </brush>
                   </colorrole>
                   <colorrole role="Dark">
                    <brush brushstyle="SolidPattern">
                     <color alpha="255">
                      <red>42</red>
                      <green>85</green>
                      <blue>127</blue>
                     </color>
                    </brush>
                   </colorrole>
                   <colorrole role="Mid">
                    <brush brushstyle="SolidPattern">
                     <color alpha="255">
                      <red>56</red>
                      <green>113</green>
                      <blue>170</blue>
                     </color>
                    </brush>
                   </colorrole>
                   <colorrole role="Text">
                    <brush brushstyle="SolidPattern">
                     <color alpha="255">
                      <red>42</red>
                      <green>85</green>
                      <blue>127</blue>
                     </color>
                    </brush>
                   </colorrole>
                   <colorrole role="BrightText">
                    <brush brushstyle="SolidPattern">
                     <color alpha="255">
                      <red>255</red>
                      <green>255</green>
                      <blue>255</blue>
                     </color>
                    </brush>
                   </colorrole>
                   <colorrole role="ButtonText">
                    <brush brushstyle="SolidPattern">
                     <color alpha="255">
                      <red>42</red>
                      <green>85</green>
                      <blue>127</blue>
                     </color>
                    </brush>
                   </colorrole>
                   <colorrole role="Base">
                    <brush brushstyle="SolidPattern">
                     <color alpha="255">
                      <red>85</red>
                      <green>170</green>
                      <blue>255</blue>
                     </color>
                    </brush>
                   </colorrole>
                   <colorrole role="Window">
                    <brush brushstyle="SolidPattern">
                     <color alpha="255">
                      <red>85</red>
                      <green>170</green>
                      <blue>255</blue>
                     </color>
                    </brush>
                   </colorrole>
                   <colorrole role="Shadow">
                    <brush brushstyle="SolidPattern">
                     <color alpha="255">
                      <red>0</red>
                      <green>0</green>
                      <blue>0</blue>
                     </color>
                    </brush>
                   </colorrole>
                   <colorrole role="AlternateBase">
                    <brush brushstyle="SolidPattern">
                     <color alpha="255">
                      <red>85</red>
                      <green>170</green>
                      <blue>255</blue>
                     </color>
                    </brush>
                   </colorrole>
                   <colorrole role="ToolTipBase">
                    <brush brushstyle="SolidPattern">
                     <color alpha="255">
                      <red>255</red>
                      <green>255</green>
                      <blue>220</blue>
                     </color>
                    </brush>
                   </colorrole>
                   <colorrole role="ToolTipText">
                    <brush brushstyle="SolidPattern">
                     <color alpha="255">
                      <red>0</red>
                      <green>0</green>
                      <blue>0</blue>
                     </color>
                    </brush>
                   </colorrole>
                  </disabled>
                 </palette>
                </property>
                <property name="autoFillBackground">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
              <item row="0" column="0">
               <widget class="QPushButton" name="calibratePushButton">
                <property name="text">
                 <string>Calibrate</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item row="0" column="0">
             <widget class="Line" name="line">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
